### PR TITLE
Readme: Grammer_errors

### PR DESCRIPTION
**Pull request by hpnightowl**

### CODE REVIEW

**Comment by hpnightowl** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 ## This is not an official Google product.
 
-Omaha is the open-source version of Google Update, a program to install requested software and keep it up to date.  The Google-branded version of Omaha is used to support software patching (both background updating, and on-demand update checks) for Google Chrome, Earth, and a variety of other Google products on Windows.
+Omaha is the open-source version of Google Update, a program to install requested software and also keep it up to date.The
+Google-branded version of Omaha is used to support software patching (both background updating, and on-demand update checks) for Google Chrome, Earth, and a variety of other Google products on Windows.
 
-We know that keeping software updated is both important and hard, and so by open-sourcing this project, our hope is that perhaps we can help others solve this problem. So, if you'd like to get involved, or even use Omaha to support your own software projects, then just follow the instructions in the [Developer Setup Guide](https://github.com/google/omaha/blob/master/doc/DeveloperSetupGuide.md), and you'll be good to go!
```
Comment (macqm) [inline README.md]: This was correct.

File: README.md [hunk 2]
```diff
@@ -2,8 +2,9 @@
 
 ## This is not an official Google product.
 
-Omaha is the open-source version of Google Update, a program to install requested software and keep it up to date.  The Google-branded version of Omaha is used to support software patching (both background updating, and on-demand update checks) for Google Chrome, Earth, and a variety of other Google products on Windows.
```
Comment (macqm) [inline README.md]: Space between `date.` and `The` removed - revert it.

**Comment by hpnightowl** [comment]
ok

**Pull request closed without merging**